sdk/src/utils/font_thumbnail.rs

```diff
@@ -41,6 +43,10 @@ const BACKGROUND_COLOR: tiny_skia::Color = tiny_skia::Color::WHITE;
 const LINE_HEIGHT_FACTOR: f32 = 1.075;
 /// How much padding to use on the left and right sides of the text
 const TOTAL_WIDTH_PADDING: f32 = 0.1;
+/// The default SVG precision
+const DEFAULT_SVG_PRECISION: u32 = 2;
```

Comment:
This needs to be gated on `add_svg_font_thumbnails` or else we get dead code warnings